doozerlib/olm/bundle.py

```diff
@@ -560,9 +560,17 @@ def redhat_delivery_tags(self):
 
     @property
     def operator_framework_tags(self):
+        version = '{MAJOR}.{MINOR}'.format(**self.runtime.group_config.vars)
+        override_channel = self.channel
+        override_default = self.channel
+        # wip see: issues.redhat.com/browse/ART-3107
+        if version == '4.10':
```

Comment:
could this be a group.yml setting? I prefer not to have things this specific in code.